requirements.txt

```diff
@@ -1,4 +1,4 @@
-mlflow
+mlflow[pipelines]>=1.27.0
 pandas==1.3.*
 pandas-profiling==3.1.*
 scikit-learn==1.0.*
```

Comment:
@jinzhang21 On second thought, I think these requirements are too strict. `mlflow[pipelines]` requires >=, generally, so users may install MLP and then have to downgrade after installing these. Can we make them looser?